setup.sh

```diff
@@ -42,29 +42,32 @@ echo    "----------------------------------------------------------------------"
 
 echo
 echo "This script will replace the above values in all files in the project."
-read -p "Are you sure you wish to continue? (n/y) " -n 1 -r
+read -p "Are you sure you wish to continue? (Y/n) " -n 1 -r
 
 echo
-if [[ ! $REPLY =~ ^[Yy]$ ]]
+if [[ $REPLY =~ ^[Nn]$ ]]
 then
     [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
 fi
 
 echo
 
-find . -type f -exec sed -i '' -e "s/:author_name/$author_name/g" {} \;
-find . -type f -exec sed -i '' -e "s/:author_username/$author_username/g" {} \;
-find . -type f -exec sed -i '' -e "s/:author_email/$author_email/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_name/$package_name/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_fullname/$package_name/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_description/$package_description/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_php_namespace/$package_php_namespace/g" {} \;
+LC_ALL=C find . -type f -exec sed -i '' -e "s/:author_name/$author_name/g" {} \;
```

Comment:
Does this have to happen before each line? Did you get a chance to try defining it once in a line above these and seeing if it applies to all?

Reply:
@mattstauffer - just tried. Looks like it has to be on each line

Reply:
OK, thanks for checking!